src/__tests__/ProductView/DetailArea.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import DetailArea from '../../ProductView/DetailArea/DetailArea.js';
+import renderer from 'react-test-renderer';
+import react from 'react-dom';
+import testProduct from '../testData/product.js';
+import testConfigurations from '../testData/frontend_configurations.js';
+let instance;
+let node = document.createElement('div');
+
+test('Render DetailArea', () => {
+    let component = react.render(<DetailArea ref={(child) => {instance = child}}
+        product={testProduct.PRODUCT}
+        configuration={testConfigurations.CONFIGURATIONS[0]}
+        showAllConfigurations={true}
+        />, node);
+    expect(instance.state.configuration).toEqual(testProduct.PRODUCT);
+});
+
+
+test('Show configuration-details', () => {
+    let component = react.render(<DetailArea ref={(child) => {instance = child}}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'component' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "component" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AOneStatementPerLine)